pkg/vpcmodel/drawioOutput.go

```diff
@@ -177,10 +183,81 @@ func (d *DrawioOutputFormatter) createEdges() {
 	}
 }
 
+// explainableEndpoints() createExplanations() to be reimplement with real Shiri work
+func (d *DrawioOutputFormatter) createExplanations() []drawio.ExplanationEntry {
+	type expKey struct {
+		src, dst EndpointElem
+	}
+	explanations := map[expKey]string{}
+	allEndpoints := d.explainableEndpoints()
+	for _, src := range allEndpoints {
+		for _, dst := range allEndpoints {
+			explanations[expKey{src, dst}] = "No Connectivity from " + src.Name() + " to " + dst.Name()
```

Comment:
We do produce explanations for the connectivity between an internal endpoint and an external entity. Does the code take care of that?

Reply:
this code do take care of that.
however, this code is about to be deleted with Shiri implementation